src/components/LaunchDetails/LaunchDetails.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Container,
+  Button,
+  Grid,
+  Card,
+  CardActionArea,
+  CardMedia,
+  CardContent,
+  CardHeader,
+  Typography,
+  ThemeProvider,
+} from '@mui/material';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { CountdownTimer } from '../CountdownTimer';
+import theme from './theme';
+import data from '../../mocks/launch-details';
+import './LaunchDetails.scss';
+
+export const LaunchDetails = () => {
+  const navigate = useNavigate();
+  const launchDate: number = Date.parse(data.net);
+  const details: string[] = [
+    'Area:',
+    `${data.pad.name}, ${data.pad.location.name}`,
+    'Status:',
+    data.status.name,
+    'Date:',
+    data.net,
+  ];
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Container className="container">
+        <Button
+          size="large"
+          variant="outlined"
+          className="goback-button"
+          onClick={() => {
+            navigate('/');
+          }}
+        >
+          Go back
+        </Button>
```

Comment:
1. `Button` should be moved to shared
2. Split this file into a tiny components

Reply:
Done.

Reply:
first item is not done

Reply:
Component moved to shared